src/tiled/mapeditor.cpp

```diff
@@ -1067,11 +1067,26 @@ EditableWangSet *MapEditor::currentWangSet() const
     return EditableWangSet::get(mWangDock->currentWangSet());
 }
 
+void MapEditor::setCurrentWangSet(EditableWangSet *wangSet)
+{
+    if (!wangSet) {
+        ScriptManager::instance().throwNullArgError(0);
+        return;
+    }
+    mWangDock->setCurrentWangSet(wangSet->wangSet());
+}
+
 int MapEditor::currentWangColorIndex() const
 {
     return mWangDock->currentWangColor();
 }
 
+void MapEditor::setCurrentWangColorIndex(int newIndex)
+{
+    if(newIndex < 0 || newIndex > mWangDock->currentWangSet()->colorCount())
+        ScriptManager::instance().throwError(QCoreApplication::translate("Script Errors", "An invalid wang color index was provided"));
```

Comment:
Should I update tiled_en.ts with this error? 
How do you keep the line numbers in that file up to date ?

Reply:
No, I will update the `tiled_en.ts` 1-2 weeks before the 1.12 release. The line numbers are not actually relevant, they just help translators find the source of a string in case it helps to see them in context.

Maybe we can include this patch in Tiled 1.11.1, but then this string just won't be translatable yet. Usually I avoid touching UI strings entirely in patch releases, but I've made exceptions for rather hidden stuff like these script errors.